pyproject.toml

```diff
@@ -240,6 +244,7 @@ server = [
     "canonicaljson",
     "jinja2",
     "jmespath",
+    "json_merge_patch",
```

Comment:
This one is also a duplicate of `json-merge-patch` in https://github.com/bluesky/tiled/blob/8c717a3555dc4f129fd1bd20946b9e3ced7602bf/pyproject.toml#L250